syntax = "proto3";
message Response {
    bool status = 1;
    string message = 2;
}
message LabelWeight {
    string image_name = 1;
    float weight = 2;
}
message ImageLabelWeight {
    string origin_image = 1;
    repeated LabelWeight label_weight = 2;
}
message GenerateImage {
    Response response = 1;
    string generate_image = 2;
}
message Action {
    string command = 1;
}
service StyleMigrate {
    rpc TrainingModel(Action) returns (Response) {}
    rpc ConvertImage(ImageLabelWeight) returns (GenerateImage) {}
}